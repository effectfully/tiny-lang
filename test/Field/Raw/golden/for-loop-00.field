for i = 0 to 0 do end; T
