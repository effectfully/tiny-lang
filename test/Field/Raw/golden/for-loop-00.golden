EStatement (EFor (Var "i") 0 0 []) (EConst (Some (UniConst Bool True)))